### ISSUE TITLE
Extraneous `|| undefined` on type-checked value causing error when calling `toString` in `toJSON`

### ISSUE THREAD

**Issue by NoahSaso** [cosmology-tech/telescope#448]
In `/osmosis.tokenfactory.v1beta1.params.Params`'s `toJSON` method, `toString` is called on a potentially undefined value:

```ts
    message.denomCreationGasConsume !== undefined && (obj.denomCreationGasConsume = (message.denomCreationGasConsume || undefined).toString());
```

```
./codegen/osmosis/tokenfactory/v1beta1/params.ts(99,85): error TS2532: Object is possibly 'undefined'.
```

`denomCreationGasConsume` is an optional bigint. Since there's the type-check at the beginning of the line, there should be no need to insert `|| undefined`.

**Comment by webmaster128** [comment]
I just ran into this too. Had to manually do this patch to make bigint work:
https://github.com/confio/cosmjs-types/pull/76/commits/6f9ba3835d06bc0c625e6953e05539e923d6062f

**Comment by Zetazzz** [comment]
Fixed in this PR:
https://github.com/cosmology-tech/telescope/pull/464

**Comment by Zetazzz** [comment]
merged and posted a PR including bigint fix and v1.0 upgrade: https://github.com/confio/cosmjs-types/pull/77
